pkg/api/controller.go

```diff
@@ -802,27 +810,76 @@ func (c *Controller) ListUsers(w http.ResponseWriter, r *http.Request, params Li
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) generateResetPasswordToken(email string) (*string, error) {
+	secret := c.Auth.SecretStore().SharedSecret()
+	currentTime := time.Now()
+	token, err := GenerateJWTResetPassword(secret, email, currentTime, currentTime.Add(DefaultResetPasswordExpiration))
+	if err != nil {
+		return nil, err
+	}
+	return &token, nil
+}
+
+func (c *Controller) inviteUserRequest(email string) error {
+	// TODO (@shimi9276) create template for sending the email with link for invite using the passed template
+	token, err := c.generateResetPasswordToken(email)
+	if err != nil {
+		return err
+	}
+	err = c.Emailer.SendEmailWithLimit([]string{email}, *token, *token, nil)
+	if err != nil {
+		return err
+	} else {
+		c.Logger.WithField("email", email).Info("invite email sent")
+	}
```

Comment:
no need to else after return - already commended about this one in the previous pr

Reply:
Fixed, missed the change in both places